config/formats.ts

```diff
@@ -676,7 +676,7 @@ export const Formats: (FormatsData | {section: string, column?: number})[] = [
 				if (!item || !item.megaStone) continue;
 				const species = this.dex.getSpecies(set.species);
 				if (species.isNonstandard) return [`${species.baseSpecies} does not exist in gen 8.`];
-				if (this.ruleTable.isRestricted('pokemon:' + species.id)) {
+				if (this.ruleTable.isRestricted(`pokemon:${species.id}`) || this.ruleTable.isRestricted(`basepokemon:${species.id}`)) {
```

Comment:
this was the fix @scheibo

Reply:
I would personally make a function that runs through the entirety of:

https://github.com/smogon/pokemon-showdown/blob/c124720885419d05f214a63ca20616390abd9299/sim/team-validator.ts#L1239-L1338

but for `*` rather than `-`.